source/features/comment-to-draft-pr-button.tsx

```diff
@@ -1,19 +1,42 @@
 import select from 'select-dom';
+import delegate from 'delegate-it';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 import onNewComments from '../github-events/on-new-comments';
 
+function handleClicks(event: delegate.Event<MouseEvent, HTMLButtonElement>): void {
+	const buttonsSingleComment = select.all([
+		'table:not([style="display: none"]) .review-simple-reply-button',
+		'table:not([style="display: none"]) .add-comment-label' // For when adding a single comment after having started a review
+	]);
+	for (const element of buttonsSingleComment) {
+		element.innerHTML = ' Add single comment to Draft PR ';
+	}
+
+	const buttonsStartReview = select.all('table:not([style="display: none"]) .start-review-label');
+	for (const element of buttonsStartReview) {
+		element.innerHTML = ' Start a review to Draft PR ';
+	}
+}
+
 function init(): void {
-	if (select.exists('#partial-discussion-header span[title="Status: Draft"]')) {
-		const commentButton = select('#partial-new-comment-form-actions > div > div.color-bg-secondary.ml-1 > button');
+	let commentButton;
+	if (pageDetect.isPRConversation()) {
+		commentButton = select('#partial-new-comment-form-actions .btn-primary');
 		commentButton!.innerHTML = 'Comment to Draft PR';
```

Comment:
```suggestion

	if (pageDetect.isPRConversation()) {
		select('#partial-new-comment-form-actions .btn-primary')!.innerHTML = 'Comment to Draft PR';
```

Can we use textContent?

Reply:
yes, but, e.g. for the Conversation tab, the 'Comment' button textContent is 
```
\n  \n  \n        Comment\n\n  \n\n
```
and so I'd have to use `.replace('Comment, Comment to Draft PR')`
instead of putting fixed string.

Are you ok with that?

Reply:
Its fine don't worry about the '\n'. I just tried it it will work correctly

Reply:
I have just tried it (in the Conversation tab), and unfortunately, the name change occurs multiple times, 
if I submit a comment(or delete comments), focus the text area and start typing:
![2021-06-27_191754](https://user-images.githubusercontent.com/723651/123551886-69731b80-d77c-11eb-899f-7c9cc7e5aabe.jpg)
<details>

<summary>Here is the code with <code>replace</code>+<code>textContent</code> instead of fixed </code>innerHTML</
code></summary>

```js
import select from 'select-dom';
import delegate from 'delegate-it';
import * as pageDetect from 'github-url-detection';

import features from '.';
import onNewComments from '../github-events/on-new-comments';

function handleClicks(): void {
	const buttonsSingleComment = select.all([
		'table:not([style="display: none"]) .review-simple-reply-button',
		'table:not([style="display: none"]) .add-comment-label' // For when adding a single comment after having started a review
	]);
	for (const element of buttonsSingleComment) {
		element.textContent = element.textContent!.replace('Add single comment', 'Add single comment to Draft PR');
	}

	const buttonsStartReview = select.all('table:not([style="display: none"]) .start-review-label');
	for (const element of buttonsStartReview) {
		element.textContent = element.textContent!.replace('Start a review', 'Start a review to Draft PR');
	}
}

function init(): void {
	let commentButton;
	if (pageDetect.isPRConversation()) {
		commentButton = select('#partial-new-comment-form-actions .btn-primary');
		commentButton!.textContent = commentButton!.textContent!.replace('Comment', 'Comment to Draft PR');
	} else if (pageDetect.isPRFiles()) {
		delegate('.diff-table', 'button', 'click', handleClicks);
	}
}

void features.add(__filebasename, {
	include: [
		pageDetect.isPRConversation,
		pageDetect.isPRFiles
	],
	exclude: [
		() => !pageDetect.isDraftPR
	],
	additionalListeners: [
		onNewComments
	],
	init
});

```

</details>

With the previous version this doesn't happen.

---

The only solution I see is to use something like this, which is apparently too easy to break:

```js
.replace('\n  \n  \n        Comment\n\n  \n\n', '\n  \n  \n        Comment to Draft PR \n\n  \n\n')
```

Reply:
Or use a regex with replace() with capturing groups around the whitespace characters.

Reply:
Yakov is not suggesting to use replace. Just use `=` and it works. The whitespace is useless

Reply:
Thanks, I just did. 

---

Btw, I feel once again embarrassed being confused over the same things over and over again..
Could I ask please to add some extra notes/warnings in https://github.com/sindresorhus/refined-github/blob/main/contributing.md to make sure people like me always follow them?

What comes to mind:

- what is the preferable way to sort imports for this repo
- never use `innerHTML` because it's flagged by the extension stores. Just use `textContent`.  
- use `select()`, not `querySelector`
- with `select()`, when having multiple selectors, split them to multiple lines
- Selectors should be short and smart/concise. Use those class and attributes combinations that are as much specific as possible and don't need almost any other specifiers.
- Use ternary `if` when possible.

Do you think these "tips" can be useful to contributors, or they are too self-explanatory/newbie level?

*I could provide a PR with these of course, but I need your opinion first.*

Reply:
Don't worry too much about these small mistakes. If the PR is salvageable you can learn about the project’s code style over time.

Some of those rules _could_ be part of the contributor guide, but I'd rather have eslint check for them. It's weird that `innerHTML` isn't already caught by eslint, e.g. 

https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-dom-node-text-content.md
https://github.com/salesforce/eslint-plugin-lwc/blob/master/docs/rules/no-inner-html.md